### PR TITLE
adding violinplot tutorial

### PR DESCRIPTION
**Pull request by XuanhaoWu**
violin plot introduction tutorial for community contribution of Yiwen Cai and Xuanhao Wu

### CODE REVIEW

**Comment by Joseph-Zhuo** [comment]
Thank you for your contribution

**Comment by jtr13** [comment]
This chapter erros since it is missing the `chick_weights.csv` file. Please follow step 6 in these instructions to add a resource file and open a new pull request to have the file merged into the project: https://jtr13.github.io/cc20/#submission-steps

**Comment by Joseph-Zhuo** [comment]
@XuanhaoWu
Hi Yiwen Cai and Xuanhao Wu, 
please follow Professor's instructions and create a new pull request when you guys are available. We could not find your chick_weights.csv file in the branch

File: introduction_to_violin_plot.Rmd [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+The choice of the kernel bandwidth will control the smoothness of the graph. Lower kernel bandwidth generates lumpier plots. However, graphs with lower kernel bandwidth perform better in identifying minor clusters. Therefore, experiments are needed in order to get the ideal kernel bandwidth used for smoothing.
+
+The ordering of the group shown in the violin plot is also important as the ordering will give insights to the graph.
+
+
+## Basic violin plot
+
+
+```{r}
+chickdata <- read.csv(file = "chick_weights.csv")
```
Comment (Joseph-Zhuo) [inline introduction_to_violin_plot.Rmd:40]: (file = "/resources/introduction_to_violin_plot/chick_weights.csv")

**Comment by XuanhaoWu** [comment]
Hi, Zhuo Zhuo
Sorry that I just see the email. We will do it soon.

**Pull request merged**